modules/oc-mirror-v2-about.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-oc-mirror-v2-about_{context}"]
+= About the oc-mirror plugin V2
+
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror all required {product-title} content and other images to your mirror registry by using a single tool. 
+
+Oc-mirror v2 is developed to address the challenges in v1, ensure a seamless experience for end users, and provide enclave support. It is available as a Tech Preview feature. To use it, add the `--v2` flag to the oc-mirror command line.
+
+oc-mirror v2 provides the following features:
+
+* A centralized method to mirror {product-title} releases, operators, and other images.
+
+* A declarative image set configuration file to include only the {product-title} releases, operators, and images that your cluster needs.
+
+* Incremental mirroring to reduce the size of future image sets.
+* Optional generation of supporting artifacts for OpenShift Update Service (OSUS) usage.
+
+* Use the existing workflow use cases from v1, including:
+- Mirror to Disk
+- Disk to Mirror
+- Mirror to Mirror
+
+* Enhance the overall user experience.
+
+* Support multi-tier complex enclaves 
+- Partial or fully air-gapped
+- Hub and spoke type air-gapped environments
+- Generate archives (tar.gz diff)
+- IDMS - ImageDigestMirrorSet
+- ITMS - ImageTagMirrorSet
+- UpdateService
+- CatalogSource
+
+* Filter operator version by bundle name
+
+* Generate mirroring archives with content selected by mirroring date
+
+* Implement delete functionality
```

Comment:
```suggestion
* Implement delete functionality
```
I don't know what this means. Maybe something like, "Several options for configuring deletion processes."